README.md

```diff
@@ -1,3 +1,86 @@
-#Exif.js
+# Exif.js
 
-A JavaScript library for reading EXIF meta data from JPEG image files.
\ No newline at end of file
+A JavaScript library for reading [EXIF meta data](https://en.wikipedia.org/wiki/Exchangeable_image_file_format) from image files.
+
+You can use it on images in the browser, either from an image or a file input element. Both EXIF and IPTC metadata are retrieved.
+This package can also be used in AMD or CommonJS environments.
+
+**Note**: The EXIF standard applies only to `.jpg` and `.tiff` images. EXIF logic in this package is based on the EXIF standard v2.2 ([JEITA CP-3451, included in this repo](/spec/Exif2-2.pdf)).
+
+## Install
+Install `exif-js` through [NPM](https://www.npmjs.com/#getting-started):
+
+    npm install exif-js --save    
+
+Or [Bower](http://bower.io/):
+
+    bower install exif-js --save
+
+Then add a `script` tag in your an HTML in the [best position](http://stackoverflow.com/questions/436411/where-is-the-best-place-to-put-script-tags-in-html-markup) referencing your local file.
+
+    <script src="vendors/exif-js/exif-js"></script>
```

Comment:
there is a type , it propably should be:
<script src="vendors/exif-js/exif.js"></script>
(The last hyphen should be a dot)